tox.ini

```diff
@@ -7,8 +7,6 @@
 setenv=
```

Comment:
Update how? Why is the section removed? Can you explain that in the commit message?